src/main/java/gmbs/util/StringUtils.java

```diff
@@ -0,0 +1,17 @@
+package gmbs.util;
+
+import java.util.stream.IntStream;
+
+public class StringUtils {
+
+    public static String[] splitByDelimiter(String string, String delimiter) {
+        return string.split(delimiter);
+    }
+
+    public static String[] stripStringArray(String[] array) {
+        int length = array.length;
+        String[] copyArr = new String[length];
+        IntStream.range(0, length).forEach(i -> copyArr[i] = array[i].strip());
+        return copyArr;
```

Comment:
```suggestion
        return Arrays.stream(array)
                .map(String::strip)
                .toArray(String[]::new);
```

Stream을 쓰는 연습을하면 코드가 더 간결해지는 경우가 있으니, 고려해보세요~